### ISSUE TITLE
high accuracy_cls, low mAP scores on training set for detection

### ISSUE THREAD

**Issue by mshu1** [facebookresearch/Detectron#561]
I used the sample "tutorial_1gpu_e2e_faster_rcnn_R-50-FPN.yaml" to train a small custom dataset with only 4k images and 100 classes. I first converted my dataset so it follows the standard COCO format. 

My image batch size is 256, and after 4000 iterations the accuracy_cls produced were around **0.82** so I terminated and used the checkpoint at iter 3999 to evaluate my results:

I used my **training** set for evaluation using test_net with the checkpoint as weight; however, my scores are the following:
AP: 0.0534
AP50: 0.1512
AP75: 0.0394
APs: 0.0103
APm: 0.0308
APl: 0.0759

I went back to check my dataset, and I don't see any problem with the format. I also used multiple checkpoints (499, 999, 1499) and did see an improvement at APs scores. The loss also dropped from 1.4 to 0.99.

However, the general scores using test_net is still quite low. Therefore, I was wondering if "accuracy_cls" is a good measurement of how well model is trained. And why we could have a high "accuracy_cls" but low AP scores? What could be done to improve the result?

Thank you!

**Comment by mamunir** [comment]
@mshu1 I also get low mAP on custom data and rest everything is fine? Have you got the solution? Thanks

**Comment by mshu1** [comment]
Hi, @mamunir 
"accuracy_cls" measures classification accuracy, so it is not an ideal measurement of how good the trained model has become. The way I did it is just keep training to decrease my cost function. I also figured that a low mAP value on my custom dataset is because of issues in my annotations, but it could vary depending on the kind of datasets you have :)
